### PR TITLE
Feature/iii 1953

### PR DESCRIPTION
**Pull request by IbenTesara**

### CODE REVIEW

File: src/styles/event-form.less [hunk 1]
```diff
@@ -775,3 +775,7 @@ form.booking-period {
 .audience .help-block {
   font-size: small;
 }
+
+.padding-left-12-5{
```
Comment (reinoutbossuyt) [inline src/styles/event-form.less]: Dergelijke utility-classes gebruiken we elders niet. Ik zou dit hier nu niet introduceren.

File: src/event_form/components/publish-modal/event-form-publish-modal.controller.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      options : {
+        minDate : today
+      }
+    };
+    vm.error = false;
+    vm.onFocus = onFocus;
+
+    function dismiss() {
+      $uibModalInstance.dismiss();
+
```
Comment (reinoutbossuyt) [inline src/event_form/components/publish-modal/event-form-publish-modal.controller.js]: vermijd dergelijke lege lijnen

File: src/event_form/components/publish-modal/event-form-publish-modal.controller.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    function publish() {
+      if (today < vm.date) {
+        eventFormData.availableFrom = vm.date;
+        $uibModalInstance.close();
+      } else {
+        vm.error = true;
+      }
+
```
Comment (reinoutbossuyt) [inline src/event_form/components/publish-modal/event-form-publish-modal.controller.js]: vermijd dergelijke lege lijnen

File: src/core/udb-place.factory.js [hunk 4]
```diff
@@ -338,6 +338,22 @@ function UdbPlaceFactory(EventTranslationState, placeCategories, UdbOrganizer) {
 
     updateTranslationState: function () {
       updateTranslationState(this);
+    },
+
+    hasFutureAvailableFrom: function() {
+      var today = new Date();
+      today = new Date(today.getFullYear(), today.getMonth(), today.getDate(), 0, 0, 0);
+      if (this.availableFrom || this.availableFrom !== '') {
```
Comment (stijnswaanen) [inline src/core/udb-place.factory.js]: I don't think availableFrom is known in this factory.

File: src/event_form/components/publish-modal/event-form-publish-modal.controller.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+   * @description
+   * # EventFormPublishModalController
+   * Modal for postponing a publish.
+   */
+  angular
+    .module('udb.event-form')
+    .controller('EventFormPublishModalController', EventFormPublishModalController);
+
+  /* @ngInject */
+  function EventFormPublishModalController($scope, $uibModalInstance, eventFormData, eventCrud, publishEvent) {
```
Comment (stijnswaanen) [inline src/event_form/components/publish-modal/event-form-publish-modal.controller.js]: $scope and eventCrud aren't used in this controller.

File: src/event_form/components/publish-modal/event-form-publish-modal.controller.js [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    var efpmc = this;
+    efpmc.error = false;
+    efpmc.hasPublicationDate = false;
+    efpmc.publicationDate = eventFormData.availableFrom;
+    var tomorrow = new Date();
+    tomorrow.setHours(0, 0, 0, 0);
```
Comment (stijnswaanen) [inline src/event_form/components/publish-modal/event-form-publish-modal.controller.js]: please use moment library here

File: src/event_form/steps/event-form-publish.controller.js [hunk 7]
```diff
@@ -46,9 +52,27 @@ function EventFormPublishController(
       });
   }
 
+  function publishLater(isEdit) {
```
Comment (stijnswaanen) [inline src/event_form/steps/event-form-publish.controller.js]: isEdit argument isn't used in this function

File: src/event_form/steps/event-form-publish.controller.js [hunk 8]
```diff
@@ -46,9 +52,27 @@ function EventFormPublishController(
...
+    var modalInstance = $uibModal.open({
+      templateUrl: 'templates/event-form-publish-modal.html',
+      controller: 'EventFormPublishModalController',
+      controllerAs: 'efpmc',
+      resolve: {
+        eventFormData: function () {
+          return controller.eventFormData;
+        },
+        eventCrud : function () {
```
Comment (stijnswaanen) [inline src/event_form/steps/event-form-publish.controller.js]: eventCrud is never used in EventFormPublishModalController, so this can be deleted

**Pull request merged**